.format.sh

```diff
@@ -3,16 +3,21 @@
 FILES=$(find src lib -iname '*.[ch]')
 
 # Format the files according to the style defined in .clang-format
-clang-format -i -style=file ${FILES} || (echo 'clang-format failed'; exit 1);
+clang-format -i -style=file "${FILES}" || (echo 'clang-format failed'; exit 1);
```

Comment:
That probably breaks in case of several files. Maybe using ``xargs`` is a better option here.

Reply:
Yes, I mentioned this in my comment